superset/connectors/druid/models.py

```diff
@@ -1559,9 +1559,9 @@ def get_filters(cls, raw_filters, num_cols, columns_dict) -> "Filter":
                     alphaNumeric=is_numeric_col,
                 )
             elif op == "IS NULL":
-                cond = Dimension(col) is None
+                cond = Filter(dimension=col, value="")
             elif op == "IS NOT NULL":
-                cond = Dimension(col) is not None
+                cond = ~Filter(dimension=col, value="")
```

Comment:
`~` is bitwise complement. You want to use `not Filter(dimension=col, value="")` instead, no?

Reply:
@betodealmeida I'm unsure. I was merely following the pattern for [`!=`](https://github.com/apache/incubator-superset/blob/master/superset/connectors/druid/models.py#L1481).

Reply:
Ah, cool, I haven't seen the rest of the file. Seems like they overloaded the method.